### PR TITLE
docs(flutter_timer): add web support

### PR DESCRIPTION
**Pull request by dawidd6**
## Status

**READY**

## Breaking Changes

NO

## Description

Adding web support for flutter_timer example.

Closes: #2334

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Comment by dawidd6** [comment]
API does not seem to work on my Chrome though. Don't know why.

**Comment by Gene-Dana** [comment]
Hi @dawidd6, it seems to work well on my end !

**Comment by dawidd6** [comment]
Hmm. Worked on desktop, but not on the web. Might have been a transient network issue maybe.

**Comment by dawidd6** [comment]
Ahhh. Posted it on wrong PR. Should be in flutter_weather.

File: examples/flutter_timer/web/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<html>
+<head>
+  <!--
+    If you are serving your web app in a path other than the root, change the
+    href value below to reflect the base path you are serving from.
+
+    The path provided below has to start and end with a slash "/" in order for
+    it to work correctly.
+
+    Fore more details:
```
Comment (felangel) [inline examples/flutter_timer/web/index.html:11]:
```suggestion
    For more details:
```

**Review by felangel — APPROVED** [review]
LGTM ✅ 
Thanks so much for the contribution! 💯 🚀 💙

**Pull request merged**